account_invoice_inter_company/models/account_move.py

```diff
@@ -33,7 +33,8 @@ def _find_company_from_invoice_partner(self):
     def action_post(self):
         """ Validated invoice generate cross invoice base on company rules """
         res = super().action_post()
-        for src_invoice in self:
+        # Intercompany account entries aren't supported
+        for src_invoice in self.filtered(lambda x: x.type != "entry"):
```

Comment:
This should be explicit to the 4 types, as there can be also other types:

https://github.com/odoo/odoo/blob/13.0/addons/account/models/account_move.py#L112

Reply:
Changes done